service/orchestrator/metrics.go

```diff
@@ -72,9 +72,9 @@ func (svc *Service) loadMetrics() (err error) {
 	// still need to be aware of the particular metric.
 	for _, m := range metrics {
 		// Somehow, we first need to save the metric, otherwise we are running into weird constraint issues.
-		err = svc.storage.Save(m, "id = ?", m.Id)
+		err = svc.storage.Create(m)
 		if err != nil {
-			log.Errorf("Error while saving metrics: %v", err)
+			log.Errorf("Error while saving metric configuration for '%s': %v", m.Id, err)
```

Comment:
I would let it as "Error while saving metrics" as we are storing the metric and not only the metric_configuration.